app/test_main.py

```diff
@@ -1,3 +1,21 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (-1, -2, [0, 0]),
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (95, 95, [19, 16]),
+    ]
+)
```

Comment:
![image](https://github.com/mate-academy/py-cat-and-dog-years/assets/155154439/9686028e-2afd-496a-87f6-13cccd8aa151)